endpoints/api/manifest.py

```diff
@@ -120,7 +130,23 @@ def get(self, namespace_name, repository_name, manifestref):
         if manifest is None or manifest.internal_manifest_bytes.as_unicode() == "":
             raise NotFound()
 
-        return _manifest_dict(manifest)
+        manifest_dict = _manifest_dict(manifest)
+
+        if parsed_args["include_modelcard"]:
+            parsed = manifest.get_parsed_manifest()
+            layer_digest = str(parsed.filesystem_layers[-1].digest)
```

Comment:
are we positive that the modelcard is always going to be in the last layer? I was looking at some of the AI images and it seems like it is the first layer. They also have annotated layer with the filename for each layer. 
```
      "annotations": {
        "org.opencontainers.image.title": "README.md"
      }
```

check the full manifest 

```
oras manifest fetch registry.redhat.io/rhelai1/granite-8b-code-instruct@sha256:3b04dfe5fcbd4bcdcd1d4185fb2645974104fb173b437efe4cea1f37518aefe5 | jq
```

Reply:
we could use annotations instead of a fixed position in the layers IMO